File: LorePdks.COMMON/DTO/Pdks/PdksDTO.cs
using LorePdks.COMMON.DTO.Base;
using LorePdks.COMMON.DTO.Firma;
using LorePdks.COMMON.DTO.FirmaCihaz;
using LorePdks.COMMON.DTO.Kisi;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LorePdks.COMMON.DTO.Pdks
{


    public class PdksDTO : BaseDTO
    {
        public FirmaDTO firmaDto { get; set; }
        public KisiDTO kisiDto { get; set; }
        public FirmaCihazDTO firmaCihazDto { get; set; }
        public DateTime? girisTarih { get; set; }
        public DateTime? cikisTarih { get; set; }
    }
}

File: LorePdks.WEB/src/app/core/models/PdksDto.ts
import { baseDto } from "./baseDto";
import { FirmaDto } from "./FirmaDto";
import { FirmaCihazDto } from "./FirmaCihazDto";
import { KisiDto } from "./KisiDto";

/**
 * Pdks bilgisini temsil eden model
 */
export interface PdksDto extends baseDto {
  firmaDto: FirmaDto;
  kisiDto: KisiDto;
  firmaCihazDto: FirmaCihazDto;
  girisTarih?: Date;
  cikisTarih?: Date;
}